design-toolkit/components/src/components/combobox/index.tsx

```diff
@@ -137,59 +72,38 @@ export function ComboBox<T extends OptionsDataItem>({
       isDisabled={isDisabled}
       isInvalid={isInvalid}
       isReadOnly={isReadOnly}
-      className='flex flex-col gap-xs'
+      className={comboBox({})}
     >
-      {({ isDisabled, isOpen }) => (
+      {({ isDisabled, isRequired }) => (
         <>
           {shouldShowLabel && (
-            <Label
-              className='empty:hidden'
-              isDisabled={isDisabled}
-              isRequired={props.isRequired}
-            >
+            <Label isDisabled={isDisabled} isRequired={isRequired}>
               {label}
             </Label>
           )}
-          <div className='relative flex items-center'>
+          <div className={input({})}>
             <Input
-              className={className}
-              isReadOnly={isReadOnly}
+              classNames={classNames?.input}
               placeholder={placeholder}
               size={size}
             />
-            <AriaButton
-              className={cn([
-                'fg-default-light absolute right-xs transform',
-                isDisabled && 'fg-disabled',
-                isOpen && 'rotate-180',
-                isReadOnly && 'hidden',
-              ])}
-            >
+            <AriaButton className={button({})}>
               <Icon size='small'>
                 <ChevronDown />
               </Icon>
             </AriaButton>
           </div>
           {shouldShowDescription && (
-            <AriaText
-              className={cn([
-                'fg-default-dark text-body-xs empty:hidden',
-                isDisabled && 'fg-disabled',
-              ])}
-              slot='description'
-            >
+            <AriaText className={descriptionText({})} slot='description'>
```

Comment:
```suggestion
            <AriaText className={descriptionText({className: classNames?.description})} slot='description'>
```
When you pass in the `classNames.description` to the "slot" style in tailwind-variants `tv` function, then it merges in whatever classNames from the application developer, allowing them to override (or augment) the styles.